src/init/kafka_exporter.rs

```diff
@@ -0,0 +1,347 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::exporters::kafka::config::{
+    AcknowledgementMode, KafkaExporterConfig, PartitionerType, SerializationFormat,
+};
+use crate::init::parse::parse_key_val;
+use clap::{Args, ValueEnum};
+use serde::Deserialize;
+
+#[derive(Debug, Args, Clone, Deserialize)]
+pub struct KafkaExporterArgs {
```

Comment:
For multiple exporter support you'll need to add the following annotation. Serde doesn't support the clap args, so any defaults will be read from the default implementation. I haven't found a way to configure clap to read from Default, since that would reduce duplication.

```
#[serde(default)]
```